modules/layout/react/lib/Layout.tsx

```diff
@@ -73,8 +73,10 @@ export default class Layout extends React.Component<LayoutProps> {
         return child;
       }
 
+      const {spacing = spacingNumbers.xs} = this.props;
```

Comment:
I'm hesitant to define the default in two places like this. What if we passed the props into this function with something like 
```tsx
{React.Children.map(children, this.renderChild.bind(undefined, this.props))}
```

Reply:
yeah, I don't love the duplication either. Will update 👍

Reply:
Turns out I can't use bind here, but I'll just make renderChild a pure function instead. 👍